app/src/main/kotlin/batect/config/Container.kt

```diff
@@ -49,7 +49,8 @@ data class Container(
     val dependencies: Set<String> = emptySet(),
     val healthCheckConfig: HealthCheckConfig = HealthCheckConfig(),
     val runAsCurrentUserConfig: RunAsCurrentUserConfig = RunAsCurrentUserConfig.RunAsDefaultContainerUser,
-    val privileged: Boolean = false
+    val privileged: Boolean = false,
+    val enableInitProcess: Boolean? = false
```

Comment:
Do we need to be able to differentiate between not set, true and false here? Otherwise I would make this non-nullable